app/views/layouts/base.html.slim

```diff
@@ -13,6 +13,8 @@ html lang="en-US"
     
     / TODO: Find a better location for this (only used in loan criteria charts)
     = javascript_include_tag "https://www.gstatic.com/charts/loader.js"
+    javascript:
+      google.charts.load('current', {'packages':['corechart']});
```

Comment:
Google Charts expects that it will be loaded only once a particular page and will return an error otherwise (at least up to the current version of Google Charts). I moved this out of the backbone view in case there are multiple breakeven question types used in a questionnaire.